test/lib/plugins/seneca-event-preloader.spec.js

```diff
@@ -0,0 +1,146 @@
+const chai = require('chai');
+const lab = require('lab').script();
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const moment = require('moment');
+const fn = require('../../../web/lib/plugins/seneca-event-preloader');
+
+const expect = chai.expect;
+chai.use(sinonChai);
+exports.lab = lab;
+// NOTE : if those test fails, you may be missing translations
+lab.describe('seneca-event-preloader', () => {
+  lab.test('should use next event date', (done) => {
+    // If you work past 23pm, blame yourself for the test not passing
```

Comment:
You couldn't have just added a day to `startTime2` and `endTime2`? 😅

Reply:
No. :p